script/Ciel/Day 8/08_01_CIEL08_4_1.txt

```diff
@@ -253,7 +253,7 @@ I don't even get to finish correcting her.
 -- アルクェイドはシュタターンと跳び去ってしまった。
 // We've clearly entered the シュターン era of sfx, this is similar to Ciel's rushing away
 // sound from the curry shop. I guess it's fast / whoosh?
-Arcueid took off like a shot.
+Arcueid took off like a bullet.
```

Comment:
"Off like a shot" is an idiom

Reply:
ref: https://dictionary.cambridge.org/dictionary/english/like-a-shot